get-poetry.py

```diff
@@ -211,7 +211,9 @@ def expanduser(path):
     from poetry.console import main
 
     main()
-"""
+""".format(
```

Comment:
I am reasonably sure hardcoding the shebang this way will break the multi-version functionality, with e.g. `pyenv`. Maybe consider keeping `/usr/bin/env` and taking the basename of `sys.executable`?